packages/manager/modules/user-contracts/src/user-contracts.constant.js

```diff
@@ -0,0 +1,5 @@
+export const CONTRACTS_LIST = ['tos', 'pp'];
```

Comment:
I think this has to come from the app... Else this makes the module specific only to the contract types of `tos` and `pp`. What do you think?